plugins/CoreAdminHome/Emails/JsTrackingCodeMissingEmail.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\Emails;
+
+use Piwik\Mail;
+use Piwik\Piwik;
+use Piwik\Site;
+use Piwik\View;
+
+class JsTrackingCodeMissingEmail extends Mail
+{
+    /**
+     * @var string
+     */
+    private $login;
+
+    /**
+     * @var string
+     */
+    private $emailAddress;
+
+    /**
+     * @var int
+     */
+    private $idSite;
+
+    public function __construct($login, $emailAddress, $idSite)
+    {
+        parent::__construct();
+
+        $this->login = $login;
+        $this->emailAddress = $emailAddress;
+        $this->idSite = $idSite;
+
+        $this->setUpEmail();
+    }
+
+    /**
+     * @return string
+     */
+    public function getLogin()
+    {
+        return $this->login;
+    }
+
+    /**
+     * @return string
+     */
+    public function getEmailAddress()
+    {
+        return $this->emailAddress;
+    }
+
+    /**
+     * @return int
+     */
+    public function getIdSite()
+    {
+        return $this->idSite;
+    }
+
+    private function setUpEmail()
+    {
+        $this->setDefaultFromPiwik();
+        $this->addTo($this->emailAddress);
+        $this->setSubject($this->getDefaultSubject());
+        $this->setReplyTo($this->getFrom());
+        $this->setWrappedHtmlBody($this->getDefaultBodyView());
+    }
+
+    private function getDefaultSubject()
+    {
+        return Piwik::translate('CoreAdminHome_MissingTrackingCodeEmailSubject', [Site::getMainUrlFor($this->idSite)]);
```

Comment:
Can we use the name instead of the URL? AFAIK a URL is optional (eg not needed for mobile apps) and also reads better maybe. I was wondering if we can incorporate "analytics" or "Matomo" somehow  into the subject but I can see it is tricky.